packages/calypso-products/src/plans-list.tsx

```diff
@@ -1491,6 +1493,7 @@ const getPlanPremiumDetails = (): IncompleteWPcomPlan => ( {
 		FEATURE_PREMIUM_THEMES_V2,
 		FEATURE_WORDADS,
 		FEATURE_STYLE_CUSTOMIZATION,
+		FEATURE_STATS_PAID,
```

Comment:
This is the checkout page.
<img width="1683" alt="Screenshot 2024-01-05 at 1 40 10 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/bf69a402-8fad-49b2-8abb-d40593b8cd8d">